File: 2022/src/A.c
#include<stdio.h>

int main() {
  int n, i;
  int Cou;

  while(1) {
    // input
    scanf("%d", &n);
    if (n == 0) {
      break;
    }
    int v[n];
    for (i = 0; i < n; i++) {
      scanf("%d", &v[i]);
    }

    // compute
    Cou = 0;
    for (i = 1; i < n - 1; i++) {
      if (v[i] > v[i-1] && v[i] > v[i+1]) {
        Cou++;
      }
    }

    // output
    printf("%d\n", Cou);
  }
  return 0;
}

File: 2022/src/A.py
while True:
    # input
    n = int(input())
    if n == 0:
        break
    vs = [*map(int, input().split())]

    # compute
    ans = 0
    for v0, v1, v2 in zip(vs, vs[1:], vs[2:]):
        if v0 < v1 and v1 > v2:
            ans += 1

    # output
    print(ans)
